tests/containers/accountLanding/AccountLandingContainer-test.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * AccountLandingContainer-test.jsx
+ * Created by Lizzie Salita 2/7/18
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+
+import * as FiscalYearHelper from 'helpers/fiscalYearHelper';
+import AccountLandingContainer from 'containers/accountLanding/AccountLandingContainer';
+
+import { mockData, mockParsed } from './mockFederalAccounts';
+
+jest.mock('helpers/accountLandingHelper', () => require('./accountLandingHelper'));
+
+// mock the child component by replacing it with a function that returns a null element
+jest.mock('components/accountLanding/AccountLandingContent', () =>
+    jest.fn(() => null));
+
+describe('AccountLandingContainer', () => {
+    it('should make an API request on mount', async () => {
+        // mount the container
+        const container = mount(<AccountLandingContainer />);
+        const parseAccounts = jest.fn();
+        container.instance().parseAccounts = parseAccounts;
+
+        await container.instance().accountsRequest.promise;
+
+        expect(parseAccounts).toHaveBeenCalledTimes(1);
+    });
+
+    describe('showColumns', () => {
+        it('should build the table', async () => {
+            // mount the container
+            const container = mount(<AccountLandingContainer />);
+
+            await container.instance().accountsRequest.promise;
```

Comment:
Do you need to wait for the API request when testing the columns?
Also, could you do a shallow mount and explicitly call `showColumns` instead?